bloc-did-method.md

```diff
@@ -8,99 +8,165 @@ The `did:bloc` DID method allows groups of independent organizations to share cu
+##### Stakeholder Config Files
+Each of these files is named `[domain].json`, where `[domain]` is the URL domain, owned by the stakeholder organization, where you can find the canonical copy of the stakeholder's configuration.
 
 A stakeholder configuration file contains:
 - The stakeholder's domain
+- The stakeholder's DID (did:bloc)
+  - TODO: also include a did:web DID doc? Same DID, same verkeys, used for bootstrapping.
 - Stakeholder custom configuration settings
 - The stakeholder's Sidetree endpoints
-- A detached JWS signature on the file, created by the stakeholder
-- a local hashlink to the previous version of this config file
+- the hash of the previous version of this config file
 
-TODO: Specify how the stakeholders are identified as the owners of the signing keys. For example, they could have DIDs, with the docs resolvable in the consortium (with some appropriate method of bootstrapping trust), or they could have x.509 certs.
+TODO: this should probably be in a secondary file, or stored in a way that's easy to detach without risk of screwing up the payload text
+- A detached JWS signature on the file, created by the stakeholder
 
 ##### History
 The `history/` directory contains historical consortium configs. Each such file is named `[hash].json`, where `[hash]` is the SHA-256 hash of the given file.
 
+TODO: instead of separate history files, have a complete history database file, as either an object or a concat JSON?
+
+TODO: consortium history may need to cache the stakeholder configs and verkeys - since these can be replaced on the stakeholder domain (or the stakeholder could disappear completely), with the associated consortium configs being subsequently unverifiable.
+
+
+##### Stakeholder Files
+    .well-known/did-bloc/
+        [domain].json
+        history/
+
+`[domain].json` is the stakeholder configuration file for this stakeholder.
+
+`history/` is a directory of previous stakeholder configs for this stakeholder
+
+TODO: is a history for the stakeholder config necessary? If the consortium caches historical stakeholder info - specifically, DID -> verkey mappings so historical consortium files can be verified, then the stakeholders don't need to keep historical data, or even exist any longer.
+
 <!--
 TODO:
 - Resolver has to check for consortium changes - how often?
   - update check frequency or cache lifetime should be part of the policy, or specified in the particular configuration file
 - when the resolver hits stakeholders to verify the consortium policy config, if it finds that a stakeholder has a newer version of a file than the consortium, how does it respond?
  - update to the latest version that has sufficient endorsement?
+ 
+- Policy may specify that a number of independent queries have to be made (queries made to endpoints 
```

Comment:
This policy would need to be enforced by the resolver.

That policy configuration maybe belongs in a separate spec.